godtools/App/Share/Data-DomainInterface/Supporting/GetTranslatedLanguageName/GetTranslatedLanguageName.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  GetTranslatedLanguageName.swift
+//  godtools
+//
+//  Created by Levi Eggert on 12/13/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+
+class GetTranslatedLanguageName {
```

Comment:
Hey @rachaelblue here is the class for sharing the business formatting for translating our app language names that are in the app languages list, downloadable languages list, and will also be used in the tool settings language list.

This was the idea I had for creating more granular classes that share business formatting, logic, etc.  It doesn't implement any interface, but can still be shared in our classes that implement domain interfaces.

I'm still not 100% sure on where to store these.  For now I placed this in /App/Share/Data-DomainInterface/Supporting/.

Reply:
Great!  Love having logic like this pulled out in a way that's easy to reuse.